style.css

```diff
@@ -54,3 +75,471 @@ p {
+  align-items: center;
+  padding-top: 3%;
+  padding-left: 0;
+}
+
+li:hover {
+  color: mediumseagreen;
+}
+
+ul.language li {
+  display: inline;
+  list-style-type: none;
+  width: fit-content;
+  justify-content: center;
+  align-items: center;
+}
+
+.projects {
+  display: inline-block;
+  margin-top: 25px;
+  background-color: #36b37f;
+  border: none;
+  color: white;
+  padding: 15px 32px;
+  text-align: center;
+  font-size: 16px;
+  justify-content: center;
+  align-items: center;
+  justify-items: center;
+  align-content: center;
+  cursor: pointer;
+}
+
+.projects:hover {
+  background-color: #172b4d;
+}
+
+.AboutMeHeader > div {
+  margin: 0;
+}
+
+.resume {
+  display: flex;
+  flex-direction: column;
+  max-width: 100%;
+  height: auto;
+  background-image: url(images/aboutme.png);
+  background-position: absolute;
+  background-size: cover;
+  background-repeat: no-repeat;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.me {
+  width: 100%;
+  height: auto;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+  font-size: 40px;
+}
+
+.help {
+  padding-left: 10%;
+  padding-right: 10%;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  color: #42526e;
+}
+
+.skills {
+  display: flex;
+  flex-direction: column;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-size: 100%;
+  line-height: 15%;
+  text-align: center;
+  color: #344563;
+  align-items: center;
+  margin-bottom: 20px;
+  padding-bottom: 5%;
+}
+
+.box1,
+.box2,
+.box3 {
+  display: grid;
+  background-color: rgba(61, 60, 61, 0.282);
+  height: 90%;
+  width: 95%;
+  gap: 10%;
+  place-items: center;
+  justify-content: stretch;
+  justify-items: center;
+}
+
+.aboutyou {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  background: url(images/illustration-contact-form.png);
+  background-size: 20%;
+  background-position: bottom right;
+  background-repeat: no-repeat;
+}
+
+.aboutyou-title {
+  font-family: 'Input', sans-serif;
+}
+
+@media screen and (min-width: 768px) {
+  .aboutyou {
+    width: 100%;
+    justify-content: space-between;
+  }
+}
+
+.comment {
+  font-size: x-large;
+  width: fit-content;
+  padding: 5%;
+  margin: 0;
+  text-align: center;
+}
+
+.form {
+  display: flex;
+  width: 90%;
+  justify-content: center;
+  justify-items: center;
+  align-items: flex-start;
+  text-align: center;
+  box-sizing: 100%;
+  padding: 10px 0 10px 0;
+  border-style: none;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+}
+
+.textarea {
+  font-family: 'Input', Arial, Helvetica, sans-serif;
+  color: slategray;
+  width: 90%;
+  padding: 10px;
+  margin-bottom: 15px;
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] [OPTIONAL] You can combine the similar media queries into just one to avoid repetition in your code